src/tool.ts

```diff
@@ -23,6 +32,148 @@ export async function printResults(tool: string, sarifFile: string) {
   endGroup()
 }
 
+export function splitStringAtFirstSlash(inputString: string | undefined): [string, string] {
+  if (inputString != null) {
+    const [firstPart, secondPart] = inputString.split('/', 2)
+    return [firstPart, secondPart]
+  }
+  return ['', '']
+}
+
+export const options: Partial<SimpleGitOptions> = {
+  baseDir: process.cwd(),
+  binary: 'git',
+  maxConcurrentProcesses: 6,
+  trimmed: false,
+}
+
+export async function prForFixSuggestion(
+  jsonFile: string,
+  fixId: string,
+  repoOwner: string,
+  repoName: string
+) {
+  let newBranch: string = 'codesec/sca/'
+
+  // git configuration
+  const git = simpleGit(options)
+  await git.addConfig('user.name', 'CodeSec Bot', false, 'global')
```

Comment:
This isn't a good customer facing name. Something like `Lacework Code Security` would be more appropriate.